index.bs

```diff
@@ -1208,6 +1208,45 @@ spec: secure-contexts; urlPrefix: https://www.w3.org/TR/secure-contexts/
 <section>
   <h2 id='examples'>Examples</h2>
 
+  <section>
+    <h3 id='example1'>Query playback capabilities with {{decodingInfo()}}</h3>
+      <p>
+        The following example shows how to use {{decodingInfo}} to query
```

Comment:
nit: Add parens to end of {{decodingInfo}} to establish the dfn link.

Reply:
Done, thanks Chris.